### PR TITLE
Migrated fragments handler library to WM beta2

### PR DESCRIPTION
**Pull request by ancirja-m**

### CODE REVIEW

**Review by soalb-m — APPROVED** [review]

File: fragmentshandler/fragmentshandler/src/androidTest/java/com/microsoft/device/dualscreen/fragmentshandler/utils/SampleActivity.kt [hunk 1]
```diff
@@ -65,30 +70,29 @@ class SampleActivity : FragmentActivity(), ScreenInfoListener {
...
-    override fun onResume() {
-        super.onResume()
-        ScreenManagerProvider.getScreenManager().addScreenInfoListener(this)
-    }
-
-    override fun onPause() {
-        super.onPause()
-        ScreenManagerProvider.getScreenManager().removeScreenInfoListener(this)
+    private fun registerWindowInfoFlow() {
+        val windowInfoRepository = windowInfoRepository()
```
Comment (CesarValiente) [inline fragmentshandler/fragmentshandler/src/androidTest/java/com/microsoft/device/dualscreen/fragmentshandler/utils/SampleActivity.kt:77]: Love it! The usage is even simpler, and requires less setup than previous version!

**Review by CesarValiente — APPROVED** [review]
LGTM! Great work Andrei!! 👏👏👏

**Comment by CesarValiente** [comment]
Wops! @ancirja-m just realized that this PR doesn't contain bumped library version and library version name in the `dependencies.gradle`. 
Please, update that before merge it. Thanks!

**Comment by ancirja-m** [comment]
> Wops! @ancirja-m just realized that this PR doesn't contain bumped library version and library version name in the `dependencies.gradle`. Please, update that before merge it. Thanks!

Done

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks for addressing these changes Andrei!

**Pull request merged**